app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Yajra\Datatables\Datatables;
+use App\Models\User;
+use App\Http\Resources\User as UserResource;
+
+class UserController extends Controller
+{
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 13</br>Opening brace should be on a new line